src/boot/efi/device-path-util.c

```diff
@@ -3,6 +3,67 @@
 #include "device-path-util.h"
 #include "util.h"
 
+EFI_STATUS make_multiple_file_device_path(
+                EFI_HANDLE device, const char16_t **files, EFI_DEVICE_PATH ***ret_dp) {
```

Comment:
if you line break then one param per line, and empty line after the bracket

Reply:
Done.